### PR TITLE
Add node driver, kontainer driver, auth provider and internal cluster info

### PR DESCRIPTION
**Pull request by vincent99**

### CODE REVIEW

File: collector/installation.go [hunk 1]
```diff
@@ -3,6 +3,7 @@ package collector
 import (
 	"github.com/satori/go.uuid"
 	log "github.com/sirupsen/logrus"
+	"regexp"
```
Comment (dramich) [inline collector/installation.go]: Nit: Should be sorted with internal at the top and 3rd party at the bottom

**Comment by dramich** [comment]
lgtm

File: collector/installation.go [hunk 2]
```diff
@@ -49,17 +59,59 @@ func (i Installation) Collect(c *CollectorOpts) interface{} {
...
 	}
 
-	// @TODO replace with unified authConfig
-	/*authConfig := "none"
-	if enabled, ok := GetSetting(c.Client, "api.security.enabled"); ok {
-		if provider, ok := GetSetting(c.Client, "api.auth.provider.configured"); ok {
-			if enabled == "true" {
-				authConfig = regexp.MustCompile("(?i)^(.*?)config$").ReplaceAllString(provider, "$1")
+	log.Debug("Collecting AuthConfigs")
+	configList, err := c.Client.AuthConfig.List(&nonRemoved)
```
Comment (rmweir) [inline collector/installation.go]: Could this be condensed as `if configList, err := c.Client.AuthConfig.List(&nonRemoved); err == nil {`?

Comment (rmweir) [reply collector/installation.go]: Same on lines 76, 89, 104

**Comment by cjellick** [comment]
Ricky LGTMs other than the stylistic change, so we'll move forward with it

**Comment by dramich** [comment]
Issue https://github.com/rancher/rancher/issues/20040

**Pull request merged**